### PR TITLE
ci: bump versions

### PR DESCRIPTION
**Pull request by ahayzen-kdab**

### CODE REVIEW

File: crates/qt-build-utils/src/lib.rs [hunk 1]
```diff
@@ -402,6 +402,14 @@ impl QtBuild {
             Err(_) => "lib",
         };
 
+        if let Ok(target) = &target {
+            if target.contains("apple") {
+                // Tell clang link to clang_rt as we build with -nodefaultlibs
+                // otherwise we cannot use helpers from the compiler in Qt
+                println!("cargo:rustc-link-arg=-fapple-link-rtlib");
```
Comment (ahayzen-kdab) [inline crates/qt-build-utils/src/lib.rs:409]: can we tell that we are building with clang here? rather than just assuming "apple" == "clang" ?

Comment (ahayzen-kdab) [reply crates/qt-build-utils/src/lib.rs:409]: Or do we always use a clang related linker on macOS ? I think yes as it's lld ?

Comment (ahayzen-kdab) [reply crates/qt-build-utils/src/lib.rs:409]: Should this be in `qt_build_utils::setup_linker` instead of `QtBuild::cargo_link_libraries` ?

Comment (ahayzen-kdab) [reply crates/qt-build-utils/src/lib.rs]: done

**Review by Montel — APPROVED** [review]

**Pull request merged**